torch/_C/__init__.pyi.in

```diff
@@ -2463,10 +2457,20 @@ def _set_worker_pids(
 def _remove_worker_pids(loader_id: _int) -> None: ...  # THPModule_removeWorkerPIDs
 def _error_if_any_worker_fails() -> None: ...  # THPModule_errorIfAnyWorkerFails
 
-# Defined in torch/csrc/DeviceAccelerator.cpp
+# Defined in torch/csrc/DeviceCapability.cpp
```

Comment:
We don't need to expose this class from C++ to Python; the detailed reasons are listed in the comments below.